simtools/psf_analysis.py

```diff
@@ -263,99 +266,99 @@ def _findPSF(self, fraction):
         """
         self._logger.debug("Finding PSF for fraction = {}".format(fraction))
 
-        xPosSq = [i**2 for i in self.photonPosX]
-        yPosSq = [i**2 for i in self.photonPosY]
-        xPosSig = sqrt(np.mean(xPosSq) - self.centroidX**2)
-        yPosSig = sqrt(np.mean(yPosSq) - self.centroidY**2)
-        radiusSig = sqrt(xPosSig**2 + yPosSig**2)
-
-        targetNumber = fraction * self._numberOfDetectedPhotons
-        currentRadius = 1.5 * radiusSig
-        startNumber = self._sumPhotonsInRadius(currentRadius)
-        SCALE = 0.5 * sqrt(currentRadius * currentRadius / startNumber)
-        deltaNumber = startNumber - targetNumber
-        nIter = 0
+        x_pos_sq = [i**2 for i in self.photon_pos_x]
+        y_pos_sq = [i**2 for i in self.photon_pos_y]
+        x_pos_sig = sqrt(np.mean(x_pos_sq) - self.centroid_x**2)
+        y_pos_sig = sqrt(np.mean(y_pos_sq) - self.centroid_y**2)
+        radius_sig = sqrt(x_pos_sig**2 + y_pos_sig**2)
+
+        target_number = fraction * self._number_of_detected_photons
+        current_radius = 1.5 * radius_sig
+        start_number = self._sum_photons_in_radius(current_radius)
+        SCALE = 0.5 * sqrt(current_radius * current_radius / start_number)
```

Comment:
Is there a reason to use uppercase for MAX_ITER, TOLERANCE, SCALE or should we also change it to snake_case?

Reply:
Presumably it is to mark these quantities as being constants (there isn't really a const in python so the convention is to write variables with uppercase to note they should be constants).